src/libaktualizr/uptane/secondaryinterface.h

```diff
@@ -19,7 +20,8 @@ namespace Uptane {
 
 class SecondaryInterface {
  public:
-  explicit SecondaryInterface(SecondaryConfig sconfig_in) : sconfig(std::move(sconfig_in)) {}
+  explicit SecondaryInterface(SecondaryConfig sconfig_in, std::shared_ptr<event::Channel> events_channel_in)
```

Comment:
Maybe I'm being pedantic, but `events_channel_in` seems misleading to me. It took me a while to understand that `_in` is here just for the syntax sake, and it's not a queue of input events (in reality it's just the opposite).

Reply:
Well, I'm killing the events_interpreter anyway, so that might be a moot point. But using `_in` to denote something just being passed in doesn't seem so bad, right?

Reply:
Well, not so bad when non-ambiguous. But if we consistenly apply the style guide (i.e. naming all class members with trailing `_`), we won't need these suffixes.